<?xml version="1.0"?>
<launch>

    <node pkg="turtlesim" type="turtlesim_node" name="turtlesim_node">
        
    </node>

    <node pkg="turtle_move" type="move_straight" name="turtle_move_straight">
        
    </node>

</launch>